code/modules/mob/living/simple_animal/hostile/megafauna/behemoth.dm

```diff
@@ -63,7 +64,7 @@
 /mob/living/simple_animal/hostile/megafauna/behemoth/OpenFire()
 	SetRecoveryTime(0, 100)
 	if(health <= maxHealth*0.5)
-		stomp_range = 2
+		stomp_range = 2 //Please do not make a 4x4 death zone that goes through walls whenever it walks... Which it does... Very quickly.
```

Comment:
Why does this need to be commented? I think this var is relatively self evident.